### DISCUSSION TITLE
What radios are the most reliable?

### DISCUSSION THREAD

**Discussion by deviantintegral** [merbanan/rtl_433#2592] [Q&A]
Over the past three years, I’ve purchased [4 NESDR Mini 2](https://www.nooelec.com/store/sdr/sdr-receivers/nesdr-mini-2.html) radios for a few different purposes. In that time:

* One radio connected to a Raspberry Pi 4 listening on 433mHz has been rock solid.
* One radio after about two years of use now resets after 10-20 minutes of use with “Async read stalled, exiting!”, and fully dropping the USB connection to the host. This happens on my Macbook too, so I’m pretty sure the dongle has died. I’m still keeping it because it’s useful for manual testing of rtl_433 on my laptop.
* I had weird issues with having two dongles connected to the same physical host (a linux NAS, so none of the typical Raspberry Pi power issues). It was like both dongles were showing as the same USB device, but it may have just been a symptom of the above dongle dying.
* On a Raspberry Pi Zero, I’ve encountered the following error and restarts with a brand new dongle:

```
rtl_433-rtl_433-1  | LIBUSB_ERROR_NOT_FOUND: Entity not found!
rtl_433-rtl_433-1  | Check your RTL-SDR dongle, USB cables, and power supply.
```

In that last case, I have a more complicated config that hops between 433 and 915 every 70 seconds. I’ve now swapped out the dongle, the USB cables, and power supply, and still have disconnects.

I’m thinking the best next step is to try a different radio. But, how should I choose?

It would be great if others could share their experiences with different available radios. If we get enough feedback, perhaps we could turn this into a wiki page or README section. For example, let’s imagine that NESDIR Mini’s have issues changing frequencies many times. We could document that as a known issue, but still recommend the radio for basic use cases given its price.

**Comment by zuckschwerdt** [comment]
- I have a NooElec NESDR SMArt running 24/7 at 1024k since 2017.
- From my ~10 "blue plastic" RTL-SDR two (somewhat) died, but the rest are now heatsinked and run at 250k only.
- Other faults: my PlutoSDR has developed a power rail problem and works only in a special configuration.

**Comment by panarom** [comment]
I don't use this phrase lightly (and here's hoping I don't jinx it 😜): these have been bullet-proof: https://www.rtl-sdr.com/buy-rtl-sdr-dvb-t-dongles/

**Reply by knarrff** [reply]
I have two of these running now for years, without any problems. They do need their power and get warm (but so do all I guess), but as long as you do take care of that, they seem to last and work well.
